_posts/2025-05-20-firefox-security-privacy-newsletter-2025-q1.md

```diff
@@ -0,0 +1,72 @@
+
+**Smartblock Embeds gives users control over blocked content**: Starting with Firefox 136, SmartBlock Embeds support empowers ETP-Strict and PBM users to consciously opt into highly visible content, and prevents the perception that the page is ‘broken’ \- even though our protections are working as intended. Smartblock works for embedded content from Instagram ([Bug 1892173](https://bugzilla.mozilla.org/show_bug.cgi?id=1892173)), TikTok ([Bug 1892172](https://bugzilla.mozilla.org/show_bug.cgi?id=1892172)) and Twitter/X ([Bug 1901602](https://bugzilla.mozilla.org/show_bug.cgi?id=1901602)), which are blocked by default in Private windows or ETP-Strict mode. Embedded blocked content from these domains will show a 1-button click option for users to un-block all content from that domain on that specific website.
+
+**Away from DNT and towards GPC**: In Q1, Firefox became the first major browser to remove the “Do Not Track” checkbox from the privacy preferences/settings, recognising that Global Privacy Control (GPC) is a better option for 2025\. See [SUMO article](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature) for details.
+
+## Core Security
+
+**Hardening Firefox:** Starting in Firefox 135, Firefox UI uses a strict Content-Security-Policy (CSP) in [browser.xhtml](https://searchfox.org/mozilla-central/source/browser/base/content/browser.xhtml) to prevent XSS-like sandbox escapes ([Bug 1935985](https://bugzilla.mozilla.org/show_bug.cgi?id=1935985)). This involved modifying over 600 inline event handlers and a careful rollout with notice to people who run modified Firefox configurations. A detailed description of these security hardening efforts are available in our blogpost: [Hardening the Firefox Frontend with Content Security Policies](https://attackanddefense.dev/2025/04/09/hardening-the-firefox-frontend-with-content-security-policies.html).
+
+**Sandboxing:** The Windows sandbox for Content Processes has been [moved to USER\_RESTRICTED](https://bugzilla.mozilla.org/show_bug.cgi?id=1403931) and rolled out in Firefox 134 and is enabled by default in Firefox 135\. This change removes read access from most of the filesystem and registry for the content process. Before, read access was generally allowed outside of the User’s home directory, which has been blocked since Firefox 56\. In Firefox 138, by [further differentiating the Gecko Media Plugin process sandbox policy](https://bugzilla.mozilla.org/show_bug.cgi?id=1952926), we were also able to enable win32k lockdown and Arbitrary Code Guard (blocking dynamic code) for the openh264 GMP process.
+
+**CSS Fuzzing:** We are also attributing a major piece of our platform goals to [Interop 2025](https://wpt.fyi/interop-2025): Enhancements in CSS syntax generation have shown immediate results and will continue to support the work throughout the year.
```

Comment:
hm... might I instead suggest:

> **CSS Fuzzing:** New CSS features are an important Firefox goal for [Interop 2025](https://wpt.fyi/interop-2025). For fuzzing, enhancements in CSS syntax generation have shown immediate improvements in coverage which will help improve the stability of these new features.

I am not sure that's actually what you were trying to say, though.